recipe/meta.yaml

```
@@ -32,7 +32,7 @@ requirements:
     - tqdm >=4.66.2,<5.0.0
     - statsmodels >=0.14.0,<0.15.0
     - dask-core >=2024.7.1,<2025.0.0
-    - python >=3.9,<3.12
+    # - python >=3.9,<3.12
```

This needs to be the same as above. You also need to bump the build number in line 17.

```suggestion
    - python >=3.9,<3.13
```